documentation/modules/mtv-performance-recommendation.adoc

```diff
@@ -174,4 +170,43 @@ Testing showed that by breaking one single large plan into multiple moderately s
 * *Maximum cold migration transfer rate of the multiple VMs (single disk) from a single ESXi host: 294 MiB/s (concurrent migration of 30 VMs, 35/50 GiB used, from Single ESXi)*
 * *Maximum cold migration transfer rate of the multiple VMs (single disk) from multiple ESXi hosts: 1173MB/s (concurrent migration of 80 VMs, 35/50 GiB used, from 8 ESXi servers, 10 VMs from each ESXi)*
 
-For additional details on performance, see xref:mtv-performance-addendum_mtv[{project-short} performance addendum]
+[id="mtv-warm-migration-recommendations_{context}"]
+== Warm migration recommendations
+
+The following are recommendations that are specific to warm migrations:
+
+[id="mtv-warm-migration-recommendations-disks-in-parallel_{context}"]
+=== Migrate up to 400 disks in parallel
+
+Testing involved migrating 200 VMs, with 2 disks each using 8 ESX hosts. All VMs were distributed evenly over the ESX hosts, 25 VMs per ESX host. During the tests, the ESX hosts monitored the total connections of `DataTransfer`. The results show each ESX host had 50 open connections for disk transfer.
+
+[id="mtv-warm-migration-recommendations-effects-of-precopy-speed_{context}"]
+=== Precopy speed drops for parallel disk migrations of over 200 disks
+
+Testing was successfully performed on parallel disk migrations with 200, 300, and 400 disks. There was a slight decrease in the precopy migration rate, approximately 25%, between the tests migrating 200 disks and those migrating 300 and 400 disks.
+
+Therefore, it is recommended to perform parallel disk migrations in groups of 300-400 disks, instead of 200 or fewer, only if a drop of 25% in precopy speed does not affect your cutover planning.
+
+[id="mtv-warm-migration-recommendations-set-cutover-time_{context}"]
+=== Set cutover time to be immediately after a migration plan starts
```

Comment:
Change to "When Possible Set the  cutover time...
As not always cut over can be set like this and customers may have struct maintenance windows